project/.travis.yml.twig

```diff
@@ -43,9 +43,9 @@ matrix:
 {% for package_name,package_versions in versions %}
 {% for version in package_versions %}
     - php: '{{ target_php|default(php|last) }}'
-      env: {{ package_name|upper }}={{ version }}.*
+      env: {{ package_name|upper }}={{ version }}{% if version !== 'dev-master' %}.*{% endif %}
```

Comment:
I mean to have a twig function which gets Version as an argument and checks internally for dev-master instead of complicating the template itself?